### ISSUE TITLE
PEP 3105 Make print a function in twisted/internet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8404]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8404|
|Type|enhancement|
|Created|2016-06-03 10:48:37Z|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8404 8404
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1464950917573940 1464950917573940
changetime__1465724104481007 1465724104481007
version__None None
owner__Amber_Brown__HawkOwl___hawkowl_____ Amber Brown (HawkOwl) <hawkowl@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/120

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl set owner to @hawkowl<br>@hawkowl set status to `closed`|
|-|-|

In changeset 1d1dc0dfdaa37427fdd38ce943ef3ed96052168e
```
#!CommitTicketReference repository="" revision="1d1dc0dfdaa37427fdd38ce943ef3ed96052168e"
Merge 8404-printfunc-internet: Make print a function in twisted.internet

Author: rodrigc
Reviewer: hawkowl
Fixes: #8404
```
